native/machine-coding/Stopwatch/style.css

```diff
@@ -0,0 +1,137 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap'); 
+
+@import url('https://fonts.googleapis.com/css2?family=Aclonica&display=swap');
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #8c52ff;
+}

```

Comment:
please import the core css as done for other challenges which comes with required styles and avoid modifying HTML and Body styles